lib/bitcoin/BitcoinProcessor.ts

```diff
@@ -545,42 +533,28 @@ export default class BitcoinProcessor {
 
     console.info(`Reverting quantile and transaction DBs to closest fee sampling group boundary given block: ${lastValidBlockNumber}`);
 
-    // For the quantile DB, we need to remove the full group (corresponding to the given
-    // lastValidBlockNumber). This is because currently there is no way to add/remove individual block's
-    // data to the quantile DB ... it only expects to work with the full groups. Which means that we
-    // need to remove all the transactions which belong to that group (and later ones).
-    //
-    // So what we need to do is:
-    // <code>
-    //   validBlockGroup = findGroupForTheBlock(lastValidBlockNumber);
-    //   firstBlockInGroup = findFirstBlockInGroup(validBlockGroup);
-    //
-    //   deleteAllTransactionsGreaterThanOrEqualTo(firstBlockInGroup);
-    //   deleteAllGroupsGreaterThanOrEqualTo(validBlockGroup);
-    // </code>
-    const firstBlockInGroup = this.getFirstBlockInGroup(lastValidBlockNumber);
+    // Basically, we need to remove all the transactions/normazlied-fee-data from the system later than the
```

Comment:
Will approve for now, and I see an opportunity to make this more consistent/readable by returning `IBlockInfo | undefined` as part of the `revertDatabases` refactoring. In fact I will do it next.

Reply:
👍